osf_tests/test_user.py

```diff
@@ -1968,3 +1970,79 @@ def test_validate_schools_bad_year(self):
             }]
             with pytest.raises(ValidationError):
                 self.user.save()
+
+
+class TestUserGdprDelete(OsfTestCase):
```

Comment:
Drive-by: New tests should use plain classes and pytest fixtures instead of OsfTestCase and unittest style.